packages/media/src/types/resource.ts

```diff
@@ -77,20 +77,13 @@ export interface Resource {
   /** Whether the resource externally hosted. */
   isExternal: boolean;
   /** Whether the resource is a placeholder. */
-  isPlaceholder?: boolean;
+  isPlaceholder: boolean;
   /** Whether the resource needs a CORS proxy. */
-  needsProxy?: boolean;
+  needsProxy: boolean;
   /** Resource creation date. */
   readonly creationDate?: string;
   /** Resource sizes */
-  sizes: { [key: string]: ResourceSize } | never[];
```

Comment:
This `never[]` was added because of the templates, where images don't have multiple sizes and instead have `"sizes": [],` (check out the template JSON files to see it)

If we change this, we should add a migration & some code to turn this into `sizes: {}` if there are no sizes or making it optional.